ci/appveyor.yml

```diff
@@ -9,9 +9,9 @@ platform: x64
 install:
   - ps: .\ci\appveyor.ps1
 build_script:
-  - "set \"JAVA_HOME=%JAVA_8_HOME%\" && mvn install -DskipTests=true -D\"maven.javadoc.skip\"=true -B -V"
```

Comment:
Not needed anymore, javadocs are only generated if the `release` profile is activated.